packages/react/src/LogoSuite/LogoSuite.figma.tsx

```diff
@@ -0,0 +1,613 @@
+import React from 'react'
+import {LogoSuite} from './LogoSuite'
+import figma from '@figma/code-connect'
+
+figma.connect(LogoSuite, 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=8553-10133&m=dev', {
+  props: {
```

Comment:
The component (specifically the `LogoSuite.Logobar`) also accepts a variant prop that's either `emphasis`, `muted`, or `undefined`.

Is that another one that's not (yet) supported in Figma?